{"Timestamp":"2020-03-10T15:18:54.6456551+07:00","Level":"Error","MessageTemplate":"{\"Identity\":\"202003100818546450818541661\",\"TransactionId\":\"\",\"Server\":\"172.16.50.127\",\"Caller\":null,\"Client\":\"\",\"Location\":{\"ClassName\":\"BranchService\",\"Namespace\":\"Pelo.v2.Web.Services.Branch\",\"Function\":\"GetAll\"},\"Message\":\"Exception of type 'Pelo.Common.Exceptions.PeloException' was thrown.\",\"Data\":{\"Parameters\":[],\"ReturnValue\":null},\"LogType\":\"Exception\",\"Elapsed\":21930,\"ExecDate\":\"2020-03-10T15:18:54.6450364+07:00\",\"Site\":null}"}
